DirectEmployers/MyJobs
# PD-3739 fixing timeline for new Site traffic report

## What's this PR do?
Fix the time line data for new site traffic report
## Screenshots (if appropriate)

## Where should the reviewer start?
Check the updated python_tld() in myreports/models.py
## How should this be manually tested?
In new site traffic report, check if values in timeline and table below reconcile
Check if the labels displayed are correct on timeline
## Any background context you want to provide?

## PR Checklist

### Front-End


### Separation of Concerns


### Migrations
N/A
